server/game/core/Player.js

```diff
@@ -424,6 +443,21 @@ class Player extends GameObject {
         return null;
     }
 
+
+    /**
+     * Returns ths top cards of the player's deck
+     * @returns {import('./card/CardTypes').PlayableCard[]} the Card,© or null if the deck is empty
+     */
+    getTopCardsOfDeck(numCard = 1) {
+        const deckLength = this.drawDeck.length;
+        const cardsToGet = Math.min(numCard, deckLength);
```

Comment:
Can we also do a contract check that numCard is > 0 (can use `assertPositiveNonZero`). Those checks have actually saved me a couple of times